packages/blade/src/components/Skeleton/_decisions/decisions.md

```diff
@@ -0,0 +1,326 @@
+# Skeleton <!-- omit in toc -->
+Skeleton Loader is a static / animated placeholder for the information that is still loading. It mimic the structure and look of the entire view. 
+
+The skeleton blocks can have 3 types of shape
+- circle 
+- rectangle
+- pill
+
+Each of these shapes can be achieved by adjusting the width, height and border radius of the blocks. Properties of skeleton loaders.
+
+<img src="./skeleton-thumbnail.png" width="380" />
+
+## Design
+- [Figma - Skeleton](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=16508%3A258522&mode=dev)
+
+
+## API
+
+| Prop          | Type        | Default     | Description                                                                                                                                             | Required |
+| ------------- | ----------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| type          | `TextTypes` | `undefined` | Predefined variants for Skeleton's dimensions when working with Typography components. (if width/height layout props are set they will take precedence) |          |
```

Comment:
As discussed, this might not be worth adding and would only lead to maintenance overheads.

Reply:
Resolved, not doing this.